deploy/examples/csi/nfs/required-rbac.yaml

```diff
@@ -0,0 +1,64 @@
+####################################################################################################
```

Comment:
nit: What about naming this rbac.yaml? This would match the pattern for deploy/examples/monitoring/rbac.yaml